assets/src/edit-story/components/panels/link/infoDialog.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useConfig } from '../../../app/config';
+
+const TwoTapFigure = styled.img`
+  min-width: 249px;
+`;
+
+const OneTapFigure = styled.img`
+  min-width: 249px;
+`;
+
+const InfoPaneContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 32px;
+`;
+const InfoPane = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+`;
+const InfoText = styled.div`
+  min-width: 180px;
+  margin-left: -48px;
+
+  b {
+    display: block;
+    margin-bottom: 12px;
+  }
+`;
+
+function LinkInfoDialog() {
+  const { pluginDir } = useConfig();
+
+  return (
+    <>
+      <span>
+        {__('Select any element ', 'web-stories')}
+        <b>
+          {__('(excluding a background or fullbleed element)', 'web-stories')}
+        </b>
+        {__(
+          ' and enter a web address. Drag your element around to convert between a 2 tap and 1 tap link.',
+          'web-stories'
+        )}
+      </span>
+      <InfoPaneContainer>
+        <InfoPane>
+          <TwoTapFigure src={`${pluginDir}assets/images/two-tap-link.png`} />
+          <InfoText>
+            <b>{__('2-Tap link', 'web-stories')}</b>
```

Comment:
We need to be more consistent with the naming here. Is it `2 tap link` or `2-tap link`? We currently use both interchangeably, but I think the latter is more correct.

Reply:
Changed to use 1-tap/2-tap